mysql-test/suite/rocksdb/t/ttl_primary_with_partitions.test

```diff
@@ -27,6 +25,7 @@ PARTITION BY LIST(c1) (
     PARTITION custom_p2 VALUES IN (3, 6, 9)
 );
 
+set global rocksdb_debug_ttl_rec_ts = -2;
```

Comment:
Might as well set these values really large or really small: -3600, or +3600. You probably want it to be two orders of magnitude different so that the changes of that much time passing is very small.